roles/openshift_control_plane/files/apiserver.yaml

```diff
@@ -0,0 +1,48 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: master-api
+  namespace: kube-system
```

Comment:
not kube-system.  I think I'd make separate namespaces for everything.

 1. etcd - has to be up very first
 2. kube-apiserver - first apiserver up, mounts etcd key
 3. kube-controller-manager - does not need etcd key availability in its namespace, skips some admission checks
 4. kube-scheduler - does not need global secret access it its namespace, skips some admission checks
 5. openshift-apiserver - while this is a unified thing it can live together.  When it is separated, bootstrapping order will demand different namespaces.  Needs etcd write key.
 6. openshift-controller-manager - Needs etcd write key.  We really should fix this and once we do it looks separate from the openshift-apiserver.  It also should not need any special bootstrapping order.

Reply:
We're going to need to sort out naming and alignment with tectonic patterns.

I'm not a huge fan of 6 separate namespaces, i'd prefer a kube control plane and an openshift control plane that we can move things out of in the future as they become less coupled.  i don't see any benefit in the etcd namespace?

Reply:
OCM shouldn't need etcd write key anymore.  That was only for the old leader code.  If you nuked that we're done with etcd access from OCM.